R/ab_scenario.R

```diff
@@ -161,33 +180,74 @@ ab_json = function(
   if(is.null(desire_lines_out$departure)) {
     desire_lines_out$departure = time_fun(n = n, ...)
   }
-  desire_lines_out$departure = desire_lines_out$departure * 10000.0
+
+  # Do not multiply by 10k if the maximum number is already greater than 7 days
```

Comment:
This sounds like we're detecting bad input, where the caller has already done this transformation. If it's an error, we should immediately fail here, so whoever's using this can fix that problem. Silently papering over it or accepting two very different types of input is likely to lead to worse problems down the line.

Reply:
Good point, failing early and asking for correct input is good. Auto fixing edge cases can be a slippery slope. Fixed in https://github.com/a-b-street/abstr/commit/f1ac2f112c525e7ee053eb37f48024e43bbea3a6